### PR TITLE
internal/db: ensure new columns are not null

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Ensure the new create_time and update_time columns are marked as not null after populating the data.
Also ensure that the data is populated safely with an updated query pattern inspired by the auth method
pattern.

### CODE REVIEW

File: internal/db/schema/migrations/oss/postgres/80/03_credential_static_base_table_updates.up.sql [hunk 1]
```diff
@@ -7,19 +7,32 @@ begin;
   alter table credential_static add column create_time wt_timestamp;
   alter table credential_static add column update_time wt_timestamp;
```
Comment (mikemountain) [inline internal/db/schema/migrations/oss/postgres/80/03_credential_static_base_table_updates.up.sql:8]: We could combine these two `alter table` commands into one, similarly to how `not null` is done lower done

File: internal/db/schema/migrations/oss/postgres/80/04_credential_library_base_table_updates.up.sql [hunk 2]
```diff
@@ -7,15 +7,27 @@ begin;
   alter table credential_library add column create_time wt_timestamp;
   alter table credential_library add column update_time wt_timestamp;
```
Comment (mikemountain) [inline internal/db/schema/migrations/oss/postgres/80/04_credential_library_base_table_updates.up.sql:8]: Ditto

File: internal/db/schema/migrations/oss/postgres/80/07_credential_store_base_table_updates.up.sql [hunk 3]
```diff
@@ -7,15 +7,27 @@ begin;
   alter table credential_store add column create_time wt_timestamp;
   alter table credential_store add column update_time wt_timestamp;
```
Comment (mikemountain) [inline internal/db/schema/migrations/oss/postgres/80/07_credential_store_base_table_updates.up.sql:8]: Ditto

**Review by mikemountain — APPROVED** [review]

File: internal/db/schema/migrations/oss/postgres/80/03_credential_static_base_table_updates.up.sql (hunk 1 above)
Comment (mgaffney) [inline internal/db/schema/migrations/oss/postgres/80/03_credential_static_base_table_updates.up.sql:8]:
```suggestion
  alter table credential_static
    add column create_time wt_timestamp,
    add column update_time wt_timestamp;
```

File: internal/db/schema/migrations/oss/postgres/80/04_credential_library_base_table_updates.up.sql (hunk 2 above)
Comment (mgaffney) [inline internal/db/schema/migrations/oss/postgres/80/04_credential_library_base_table_updates.up.sql:8]:
```suggestion
  alter table credential_library
    add column create_time wt_timestamp,
    add column update_time wt_timestamp;
```

File: internal/db/schema/migrations/oss/postgres/80/07_credential_store_base_table_updates.up.sql (hunk 3 above)
Comment (mgaffney) [inline internal/db/schema/migrations/oss/postgres/80/07_credential_store_base_table_updates.up.sql:8]:
```suggestion
  alter table credential_store
    add column create_time wt_timestamp,
    add column update_time wt_timestamp;
```

**Review by mgaffney — APPROVED** [review]

**Pull request merged**